test/statsd_test.rb

```diff
@@ -52,7 +52,7 @@ def test_statsd_measure_with_explicit_value_and_sample_rate
   end
 
   def test_statsd_measure_with_benchmarked_block_duration
-    StatsD::Instrument.stubs(:current_timestamp).returns(5.0, 5.0 + 1.12)
+    Process.stubs(:clock_gettime).returns(5.0, 5.0 + 1.12)
```

Comment:
❤️ thanks for fixing this up so it mocks true dependencies and not internal wiring of the code.

Reply:
Are we doing this in applications?  Since it can result in unintended consequences by mocking something more generic.

Reply:
After discussing with @dylanahsmith the concern was around Shopify applications and unrelated to this public gem. There isn't an issue with this stubbing as it is in a _unit test_ for a piece of logic where the appropriate seam to stub is `Process.clock_gettime`